vrayGolaem/vraygolaem.cpp

```diff
@@ -182,10 +182,10 @@ static ParamBlockDesc2 param_blk(params, STR_DLGTITLE,  0, &vrayGolaemClassDesc,
 	p_default, TRUE,
 	p_ui, TYPE_SINGLECHEKBOX, ED_DISPLAYENABLE,
 	PB_END,
-	pb_display_percent, _T("display_percent"), TYPE_INT, P_RESET_DEFAULT, 0,
-	p_default, 100,
-	p_range, 0, 100, 
-	p_ui, TYPE_SPINNER,  EDITTYPE_POS_INT, ED_DISPLAYPERCENT, ED_DISPLAYPERCENTSPIN, 1,
+	pb_display_percentage, _T("display_percentage"), TYPE_FLOAT, P_RESET_DEFAULT, 0,
```

Comment:
Changing the name will break any existing MaxScripts that use this parameter. Are you ok with this?